charts/rancher-vsphere-csi/questions.yaml

```
@@ -94,7 +94,14 @@ questions:
     type: boolean
     default: true
     group: Driver Configuration
-
+    
+  - variable: csiInternalGeneratedClusterId.enabled
+    label: Enable Internally Generated Cluster ID
+    description: Enables the vSphere CSI Driver to auto-generate cluster IDs. The CSI driver will generate and persist a unique cluster ID using a ConfigMap.
+    type: boolean
+    default: true
```

I think this should be off by default, unless you can explain why we would want to suddenly turn it on for everyone.
```suggestion
    default: false
```